### PR TITLE
feat(core/css/respec): add :target highlight #1683

### PR DESCRIPTION
**Pull request by Jigar3**
#1683 
Added a little CSS on :target which lets one focus easily where the hash link lead to.
Here is the screenshot.

screenshot from 2018-06-28 23-56-07

### CODE REVIEW

File: js/core/css/respec2.css [hunk 1]
```diff
@@ -39,6 +39,13 @@ a.bibref {
   text-decoration: none;
 }
 
+a.bibref:target {
```
Comment (sidvishnoi) [inline js/core/css/respec2.css]: We may want the `:target` css to be applied to more places, not just `a.bibref`

Comment (Jigar3) [reply js/core/css/respec2.css]: so i should just change a.bibref:target to target??

Comment (sidvishnoi) [reply js/core/css/respec2.css]: @marcoscaceres WDYT?

Comment (sidvishnoi) [reply js/core/css/respec2.css]: Also, we may want to consider Wikipedia like `:target` styling. It's simpler :slightly_smiling_face:

Comment (Jigar3) [reply js/core/css/respec2.css]: hey just set the background-color to #eaf3ff :smile:

Comment (sidvishnoi) [reply js/core/css/respec2.css]: Lets do that :smiley:

Comment (Jigar3) [reply js/core/css/respec2.css]: :+1:  on all :target

**Comment by Jigar3** [comment]
Updated Screenshot 
screenshot from 2018-06-29 00-48-15

**Comment by sidvishnoi** [comment]
Oops. found an issue (entire section getting higlighted).
image

What do you think we should do? Use some JS? @marcoscaceres

**Comment by Jigar3** [comment]
@sidvishnoi how to test my changes?. Actually i was just changing styles in dev tools.

File: js/core/css/respec2.css [hunk 2]
```diff
@@ -39,6 +39,10 @@ a.bibref {
   text-decoration: none;
 }
 
+#references:target {
```
Comment (sidvishnoi) [inline js/core/css/respec2.css]: expected selector: `#references :target`
typo? :slightly_smiling_face:

**Pull request merged**